gitCore/api/src/main/java/com/virtuslab/gitcore/api/IGitCoreBranchSnapshot.java

```diff
@@ -40,11 +39,11 @@ static boolean defaultEquals(@FindDistinct IGitCoreBranchSnapshot self, @Nullabl
     } else {
       val o = (IGitCoreBranchSnapshot) other;
       return self.getFullName().equals(o.getFullName())
-          && Try.of(() -> self.getPointedCommit().equals(o.getPointedCommit())).getOrElse(false);
```

Comment:
why was it try?

Reply:
I'd assume that `getPointedCommit()` originally could throw an exception... @MaciejG604 if you could check what was the original implementation (`git bisect` might be useful, given the size of the history)

Reply:
Commit 93bb80728a17c285e11a36708ee44583c1d6ad2a

Reply:
Gr8... so indeed the `throws` clause has been dropped from the method, but `Try.of` unnecessarily retained... makes sense